main.py

```diff
@@ -123,8 +133,9 @@ def ensure_conn_secret(kube, namespace, secret_name, conn):
 @click.option('--db-name', envvar='DB_NAME', required=True)
 @click.option('--secret-name', envvar='SECRET_NAME', required=True)
 @click.option('--namespace', envvar='NAMESPACE', required=True)
+@click.option('--is-conn-string', envvar='IS_CONNECTION_STRING', type=bool, default=True)
```

Comment:
```suggestion
@click.option('--conn-as-separate-fields', envvar='IS_CONNECTION_STRING', type=bool, default=True)
```

I think is a slightly more descriptive name for this.

Reply:
done.

Reply:
@ashb @andriisoldatenko the flag is actually corresponding to https://github.com/astronomer/astronomer/pull/566/files.

I don't have too much of a preference, but thinking we should use a consistent naming scheme through both places so its easier to trace and reason about.